### PR TITLE
Add shared-modules section to handbook->javascript

### PR DESCRIPTION
**Pull request by sharvit**
Add documentation about shared node-modules and the `config/webpack.vendor.js`

### CODE REVIEW

File: handbook.md [hunk 1]
```diff
@@ -152,6 +152,12 @@ We use eslint to enforce linting rules. Run eslint by typing `npm run lint` in t
 * Pollute the global namespace. If, for some reason, you must expose a function/value globally, use the ```window.tfm``` object as implemented in ```bundle.js```.
 * Remove global functions from existing code without deprecating them first. Deprecation can be done by calling the `tfm.tools.deprecate` function.
 
+#### Shared modules
+Usually, when we add a new node-module, we want it to be shared with plugins (Unless we have a reason not share it).
```
Comment (dLobatog) [inline handbook.md]: `a reason not to share it`

File: handbook.md [hunk 2]
```diff
@@ -152,6 +152,12 @@ We use eslint to enforce linting rules. Run eslint by typing `npm run lint` in t
...
+
+In order to do so, we let [webpack](https://webpack.js.org/) to create a `vendor.js` file with the relevant modules.
```
Comment (dLobatog) [inline handbook.md]: `we let webpack create`

**Review by dLobatog — CHANGES_REQUESTED** [review]
Great addition, thanks @sharvit

**Comment by sharvit** [comment]
Thanks @dLobatog, updated!

**Review by dLobatog — APPROVED** [review]
Merging! :medal_military: @sharvit

**Pull request merged**